setup.py

```diff
@@ -32,6 +31,10 @@
 ext_modules = []
 
 if "--pyprof" in sys.argv:
+    try:
+        from pip._internal import main as pipmain
```

Comment:
`19.3.1` will be able to import it without an `ImportError`, so we need to call `pipmain` and catch the `TypeError` instead.

Reply:
This should work:
```python
if "--pyprof" in sys.argv:
    try:
        from pip._internal import main as pipmain
        pipmain() # dummy call to raise the TypeError for pip >= 19.
    except TypeError:
        from pip._internal.main import main as pipmain
```